wcivf/apps/elections/templates/elections/includes/_city_of_london_registration_details.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+
+<li>
+    <details open>
+        <summary>
+            <h2>
+                <span aria-hidden="true"></span>
+                {% blocktrans trimmed %}Register to vote{% endblocktrans %}
+            </h2>
+        </summary>
+        <p>
+            {% blocktrans trimmed %}
+                City of London council elections do not use the same electoral register as other elections.
+            {% endblocktrans %}
+        </p>
+        <p>
+            {% blocktrans trimmed %}
```

Comment:
I wrapped all this text up for translation, but I don't really think there is a big rush to get it translated given this is text we would only ever show to users in England